streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -137,6 +137,8 @@
     private final boolean eosEnabled;
     private final static long DEFAULT_COMMIT_INTERVAL_MS = 30000L;
     private final static long EOS_DEFAULT_COMMIT_INTERVAL_MS = 100L;
+    private final static String NO_OPTIMIZATION = "none";
+    private final static String OPTIMIZE = "all";
```

Comment:
Should we make both public similar to `UPGRADE_FROM_0100` or `AT_LEAST_ONCE` ?

Reply:
ack